model/clusters_mgmt/v1/node_pools_resource.model

```diff
@@ -24,6 +24,37 @@ resource NodePools {
         // Number of items contained in the returned page.
         in out Size Integer = 100
 
+        // Search criteria.
+		// 
+		// The syntax of this parameter is similar to the syntax of the _where_ clause of a
+		// SQL statement, but using the names of the attributes of the cluster instead of
```

Comment:
it would be good to have a real example valid for node pools, like we do here https://github.com/openshift-online/ocm-api-model/blob/main/model/clusters_mgmt/v1/pending_delete_clusters_resource.model#L29